### PR TITLE
Add support for cancel shutdown for virtual controller

### PR DESCRIPTION
**Pull request by svt-vverma**
Signed-off-by: Vansh Verma <vansh.verma@hpe.com>

### Description
Add support for cancel shutdown for virtual controller

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/hosts.py [hunk 1]
```diff
@@ -180,3 +180,16 @@ def get_virtual_controller_shutdown_status(self):
...
         return self._client.do_get(resource_uri)
+
+    def cancel_virtual_controller_shutdown(self, timeout=-1):
+        """Cancels the virtual controller shutdown.
+
+        Args:
+          timeout: Time out for the request in seconds.
+
+        Returns:
+          object: Response object.
```
Comment (svt-kmcdaniel) [inline simplivity/resources/hosts.py]: I don't think this is returning an object, isn't it just a `dict` ?

Comment (svt-kmcdaniel) [reply simplivity/resources/hosts.py]: @sijeesh should we just return the status string?

Possible values are "STRING ['SUCCESS', 'FAILURE', 'UNKNOWN', 'IN_PROGRESS' ]

**Review by svt-kmcdaniel — CHANGES_REQUESTED** [review]

File: simplivity/resources/hosts.py (hunk 1 above)
Comment (svt-vverma) [reply simplivity/resources/hosts.py]: Updated as per suggestion returning the string response now

**Review by sijeesh — APPROVED** [review]

File: simplivity/resources/hosts.py (hunk 1 above)
Comment (sijeesh) [reply simplivity/resources/hosts.py]: @svt-kmcdaniel , Yes, a status string should be fine here.

**Review by svt-kmcdaniel — APPROVED** [review]

**Comment by sijeesh** [comment]
@svt-vverma, please resolve the conflicts to merge the PR.

**Comment by svt-vverma** [comment]
@sijeesh Resolved the merge conflicts.

**Pull request merged**